### PR TITLE
Adding z-version upgrade for all architectures for OCP 4.15

### PR DESCRIPTION
**Pull request by JianLi-RH**
Replace https://github.com/openshift/release/pull/47915

/cc @jianlinliu @shellyyang1989 @liangxia @jiajliu

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.15.yaml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EDGE_ZONE_TYPE: wavelength-zone
+      FIPS_ENABLED: "true"
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.15 and @aws-ipi'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.15 and @aws-ipi'
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-localzone
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.15.yaml:102]: the workflow is being updated to `cucushift-installer-rehearse-aws-ipi-edge-zone`

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.15.yaml:102]: @JianLi-RH https://github.com/openshift/release/pull/48418 just got merged, please rebase the code.

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.15.yaml]: Done. @jianlinliu

**Comment by liangxia** [comment]
ci/prow/core-valid — Job failed.

**Comment by liangxia** [comment]
The log show there are conflicts. Perhaps try another rebase ?
```
CONFLICT (content): Merge conflict in ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml\nAutomatic merge failed; fix conflicts and then commit the result.
```

**Comment by smg247** [comment]
/hold cancel
Now we can see the `core-valid` test is failing

**Pull request merged**